Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.h

```diff
@@ -101,8 +100,14 @@ namespace Multiplayer
         void SendEditorServerLevelDataPacket(AzNetworking::IConnection* connection) override;
         //! @}
 
+        //! AZ::TickBus::Handler
+        //! @{
+        void OnTick(float, AZ::ScriptTimePoint) override;
+        //! @}
+
         IEditor* m_editor = nullptr;
-        AzFramework::ProcessWatcher* m_serverProcess = nullptr;
+        AzFramework::ProcessWatcher* m_serverProcessWatcher = nullptr;
```

Comment:
What deletes m_serverProcessWatcher?

Reply:
This is a memory leak.
The m_serverProcessWatcher should be stored in a unique_ptr

Reply:
Second the unique_ptr recommendation

Reply:
nice catch!